### ISSUE TITLE
Documentation pull request. "Installation and Usage"

### ISSUE THREAD

**Issue by Squelch** [OpenFactorioServerManager/factorio-server-manager#404]
I have made some revisions to the Installation and Usage wiki page to clear up a few ambiguities and misunderstandings that seem to have occured. I have also elaborated of some other instructions to clarify login details.

Please see the revisions on my clone of the wiki:
https://github.com/Squelch/factorio-server-manager/wiki/Installation-and-Usage

**Comment by Mattie112** [comment]
> Visit localhost:8080 in your web browser.

Perhaps change to:
Visit `http://localhost:8080` in your web browser.

(And also: is the port correct? Earlier we are binding port 80 to port 80)

Perhaps it would also be best to use a volume mapping in the first example. Or at least refer to where your saves etc are stored to prevent people from losing them. Or at least hint to the more elaborate docker instructions at https://github.com/OpenFactorioServerManager/factorio-server-manager/tree/develop/docker#readme

I would also suggest to add somekind of FAQ (https://github.com/OpenFactorioServerManager/factorio-server-manager/issues/389) to at least address the "I forgot my password" part :)

Other then that: looks good to me!

edit: my comment about the ports mismatches docker and linux/windows, nevertheless the port is 80, not 8080
https://github.com/OpenFactorioServerManager/factorio-server-manager/blob/develop/src/bootstrap/config.go#L23

**Comment by Squelch** [comment]
The port number is indeed inconsistent between platforms, and is intended to be the subject of a future issue.

A docker map would be useful, and my thought was it could be added to a Docker specific page for those that need it, and some basic commands for those that are not expert with using rather than developing on Docker. There's a few other inconsistencies with how the Docker image performs compared to other platforms that might need addressing or documenting too.

I have also been contemplating adding a Password heading to the page to recap the instructions on where to find the initial password, and what to do when the password is lost. Thanks for confirming my thoughts there.

These revisions are an inital start to make some clarifications and avoid confusion in the first instance. Further expansion of the documentation overall would be nice, but it is probably prudent to wait until 1.1 is close to include new features and changes.

I do not have edit rights on this wiki, hence the need to have cloned it and make edits on my own copy. All revisions I've made are commented, so it would be possible to either merge the repos, or simply copy/paste from the history where required. I will continue to develope further pages on my copy in the meantime as work in progress. I would invite anyone wishing to collaborate to join me.

**Comment by Mattie112** [comment]
Yeah it is a bit strange that you cannot make a PR for the wiki in github.... Would make it easier to change some smaller things.

But nice work at least starting with it ;)

**Comment by Squelch** [comment]
Further revisions made to incorporate suggestions made by @Mattie112 

1. Link to further Docker image information
2. Login and Password section added
3. Subsection detailing login recovery
4. Added note that Factorio Server is included in the docker container
